apertium-swe.swe.dix

```diff
@@ -117922,7 +117923,7 @@ Current differences from other Apertium dictionaries:
 <e lm="tape"><i>tap</i><par n="tap/e__n_ut"/></e>
 <e lm="Jupitermåne"><i>Jupitermån</i><par n="ving/e__n_ut"/></e>
 <e lm="abborre"><i>abborr</i><par n="ving/e__n_ut"/></e>
-<e lm="aborre"><i>aborr</i><par n="ving/e__n_ut"/></e>
+<e lm="aborre">r="LR" c="common misspelling" a="PT"<i>aborr</i><par n="ving/e__n_ut"/></e>
```

Comment:
the attributes should be inside the element, e.g.

`<e lm="aborre" r="LR" c="common misspelling" a="PT"><i>aborr</i><par n="ving/e__n_ut"/></e>`